tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -34,10 +34,10 @@ static int getSortKey(float min_zoom, int population_rank, long population, Stri
       .orderByInt((int) min_zoom * 10, 0, 150)
       // population_rank DESC NULLS LAST,
       //population_rank.isEmpty() ? 15 : population_rank.getAsInt()
-      .thenByInt(population_rank, 0, 15)
+      .thenByInt(population_rank, 15, 0)
```

Comment:
We need to invert this: 15 population rank should be the highest priority (lowest sort key). same with population below